php/example_code/secretsmanager/AttachLabel.php

```diff
@@ -1,74 +1,63 @@
- * specific language governing permissions and limitations under the License.
- *
- *  ABOUT THIS PHP SAMPLE: This sample is part of the KMS Developer Guide topic at
- *  https://docs.aws.amazon.com/kms/latest/developerguide/programming-key-policies.html
- *
- */
-// snippet-start:[secretsmanager.php.attach_label.complete]
-// snippet-start:[secretsmanager.php.attach_label.import]
-
-require 'vendor/autoload.php';
-
-use Aws\SecretsManager\SecretsManagerClient; 
-use Aws\Exception\AwsException;
-// snippet-end:[secretsmanager.php.attach_label.import]
-
-/**
- * Add a staging label to a version of an AWS Secret Manager Secret.  
- *
- * Use ListSecretVersions.php to identify the VersionID for a Secret.
- *
- * This code expects that you have AWS credentials set up per:
- * https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/guide_credentials.html
- */
-
-//Create a Secrets Manager Client 
-// snippet-start:[secretsmanager.php.attach_label.main]
-$client = new SecretsManagerClient([
-    'profile' => 'default',
-    'version' => '2017-10-17',
-    'region' => 'us-west-2'
-]);
-
-$secretName = '<<{{MySecretName}}>>';
-$version_tag = 'AWSCURRENT';
-$version_id = 'EXAMPLE1-90ab-cdef-fedc-ba987SECRET1';
-
-try {
-    $result = $client->updateSecretVersionStage([
-        'VersionStage' => $version_tag,
-        'SecretId' => $secretName,
-        'MoveToVersionId' => $version_id,
-    ]);
-    var_dump($result);
-} catch (AwsException $e) {
-    // output error message if fails
-    echo $e->getMessage();
-    echo "\n";
-}
- 
-// snippet-end:[secretsmanager.php.attach_label.main]
-// snippet-end:[secretsmanager.php.attach_label.complete]
-// snippet-comment:[These are tags for the AWS doc team's sample catalog. Do not remove.]
-// snippet-sourcedescription:[AttachLabel.php demonstrates how to add a label to a version of a secret in AWS Secrets Manager.]
-// snippet-keyword:[PHP]
-// snippet-sourcesyntax:[php]
-// snippet-keyword:[AWS SDK for PHP v3]
-// snippet-keyword:[Code Sample]
-// snippet-keyword:[AWS Secrets Manager]
-// snippet-service:[secretsmanager]
-// snippet-sourcetype:[full-example]
-// snippet-sourcedate:[2018-11-08]
-// snippet-sourceauthor:[jschwarzwalder (AWS)]
\ No newline at end of file
+<?php
+
+/**
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * This file is licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License. A copy of
+ * the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ *  ABOUT THIS PHP SAMPLE: This sample is part of the KMS Developer Guide topic at
+ *  https://docs.aws.amazon.com/kms/latest/developerguide/programming-key-policies.html
```

Comment:
This is the KMS Developer guide, should it be the SecretsManager guide referenced here?

Reply:
Good catch … I think those references should be removed all together even though they [those links don’t show up in how the examples are currently presented](https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/secretsmanager-examples-manage-secret.html). If these do get move to the SOS code examples, there will be links to the GitHub repo where people will see them.

If that link is followed, there is wrong and I don't see that content in the Secrets Manager guide.

Reply:
Should I make the changes and commit them or is that something you'll be doing?

Reply:
If you could make them, that would help me keep going on other PRs.

Reply:
Changes made.